demo/api/src/products/entities/product-category.entity.ts

```diff
@@ -34,7 +34,7 @@ export class ProductCategory extends BaseEntity<ProductCategory, "id"> {
         //search: true, //not implemented
         //filter: true, //not implemented
         //sort: true, //not implemented
-        input: false, //default is true
+        input: true, //default is true
```

Comment:
This change seems unrelated to me... is this PR based on another PR?

Reply:
I had to change the api to get an update-mutation suited to change multiple product-category assignments: https://github.com/vivid-planet/comet/pull/2449/commits/d01cd3841357292fc66ca81ffc073b8a4e172bb1